src/LiveSplit.AutoSplittingRuntime/ASR.cs

```diff
@@ -667,6 +669,7 @@ internal Widgets(IntPtr ptr) : base(ptr) { }
 
     public delegate int StateDelegate();
     public delegate void SetGameTimeDelegate(long gameTime);
+    public delegate void SetCustomVariableDelegate(IntPtr namePtr, UIntPtr nameLen, IntPtr valuePtr, UIntPtr valueLen);
```

Comment:
Do not place delegates inside classes.

Is passing a `Func<IntPtr, UIntPtr, IntPtr, UIntPtr>` as `set_custom_variable` not possible?

Reply:
Now that the other delegates have been moved outside, I've moved `SetCustomVariableDelegate` outside as well, next to them.